### ISSUE TITLE
MessageContentUnion is empty

### ISSUE THREAD

**Issue by Fxlr8** [airgram/airgram#119]
**Issue type:**

[ ] bug report

**Airgram version:**

[ ] `latest`

<img width="840" alt="image" src="https://user-images.githubusercontent.com/10725007/87874184-80d45100-c9d0-11ea-9afc-1aca3aff1d39.png">

Hi, looks like message.content type has no properties. 
I think this happens because currently it is a `union` of all possible types.
```js
MessageContentUnion = MessageText | MessageAnimation | MessageAudio...
```
I think it should be an aggregate of all possible types. 
```js
MessageContentUnion = MessageText & MessageAnimation & MessageAudio...
```

**Comment by esindger** [comment]
Hello,
I believe current behavior is correct. You can use chaining operator to avoid the error. Like:
```ts
const text = update.message.content?.text
```

**Comment by Fxlr8** [comment]
Nope, looks like there is a problem with types. `text` property of `content` should be optional, but it does not exist.

<img width="736" alt="image" src="https://user-images.githubusercontent.com/10725007/88546635-56644280-d025-11ea-8833-573c904358ed.png">

**Comment by esindger** [comment]
The `text` property is not optional. Some content types (like `messageText`) always contain this property, while others never do. Different message types represent by different set of properties and almost all properties are "optional".

You need to check what kind of content you handle to avoid this error:
```ts
airgram.on('updateNewMessage', async (ctx) => {
      const { message } = ctx.update
      if (message.content._ === 'messageText') {
        console.log(message.content.text) // no error
      }
})
```

**Comment by Fxlr8** [comment]
I did exactly what you said, but I still get the error. What version of typescript are you using?

<img width="1057" alt="image" src="https://user-images.githubusercontent.com/10725007/89102337-ec76df00-d410-11ea-855c-b3d62dfb888c.png">

**Comment by esindger** [comment]
I've tested with Typescript `3.7.5` and `3.9.7`.
Can you create a small repository to reproduce this error?
